movies.py

```diff
@@ -0,0 +1,100 @@
+"""Movie ticket process system and database"""
+
+import pprint
+
+MOVIE_PRICE = 5.0
+
+def customer_db():
```

Comment:
This and the movie_db() function should really only be used in your testing module.